jni/src/knn_extension/faiss/utils/BitSet.cpp

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "BitSet.h"
+
+FixedBitSet::FixedBitSet(const int* intArray, const int length){
+    int maxValue = intArray[length - 1];
```

Comment:
Will this always be ordered? Can we add that in comment in header file? Also, do we need to do any empty/null checks?

Reply:
It will be ordered always. It shouldn't be empty or null. Let me add the check.